ArduCopter/mode_zigzag.cpp

```diff
@@ -0,0 +1,354 @@
+#include "Copter.h"
+
+/*
+* Init and run calls for zigzag flight mode
+*/
+
+#define ZIGZAG_RANGE_AS_STATIC  3
+
+// zigzag_init - initialise zigzag controller
+bool Copter::ModeZigzag::init(bool ignore_checks)
+{
+    if (copter.position_ok() || ignore_checks) {
+        // initialize's loiter position and velocity on xy-axes from current pos and velocity
+        loiter_nav->init_target();
+
+        // initialize vertical speed and acceleration's range
+        pos_control->set_speed_z(-get_pilot_speed_dn(), g.pilot_speed_up);
+        // the parameters are maximum climb and descent rates
+        pos_control->set_accel_z(g.pilot_accel_z);
+        // Global parameters are all contained within the 'g' class.
```

Comment:
So this part is not needed? It seems a lot has changed about flight modes since a few months ago when we wrote this mode. So I have some catch-up work to do :)

Reply:
I really should have said "dead, commented-out ode".  This comment and the commented-out `Parameters g` should be removed.